Description: Cast correctly to uintmax_t when using %PRIxMAX.
 I'll forward this patch as soon as I catch up with the dma upstream.
Origin: other: http://svn.ringlet.net/svn/ringlet/mail/dma/
Forwarded: no
Author: Peter Pentchev <roam@ringlet.net>
Last-Update: 2010-06-02

--- a/spool.c
+++ b/spool.c
@@ -89,7 +89,7 @@
 	 */
 	if (fstat(fd, &st) != 0)
 		goto fail;
-	if (asprintf(&queue->id, "%"PRIxMAX, st.st_ino) < 0)
+	if (asprintf(&queue->id, "%"PRIxMAX, (uintmax_t)st.st_ino) < 0)
 		goto fail;
 
 	queue->mailf = fdopen(fd, "r+");
